src/components/v2v3/V2V3Project/V2V3FundingCycleSection/FundingCycleDetails/FundingCycleListItems/index.tsx

```diff
@@ -29,9 +30,18 @@ export function FundingCycleListItems({
     distributionLimitCurrency: oldDistributionLimitCurrency,
   } = useContext(V2V3ProjectContext)
 
-  const formattedStartTime = fundingCycle.start
+  // Required for reconfigure review
+  const wouldStartImmediately =
+    !oldFundingCycle || oldFundingCycle.duration.eq(0)
+
+  const formattedStartTime = wouldStartImmediately
+    ? t`Immediately`
+    : fundingCycle.start
     ? formatDate(fundingCycle.start.mul(1000))
     : undefined
+
+  const hasAlreadyStarted = fundingCycle.start.lt(BigNumber.from(moment.now()))
```

Comment:
**suggestion** avoid moment whereever possible, library is fat AF, could this simply eb `Date.now()` instead?